processor/throughputmeasurementprocessor/metrics.go

```diff
@@ -0,0 +1,59 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package throughputmeasurementprocessor
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opentelemetry.io/collector/obsreport"
+)
+
+const tagProcessorKey = "processor"
```

Comment:
This may just be my lack of knowledge, but just commenting so I can learn something.

As far as the stats package is concerned, the TagKeys are described as this: 

```
	// TagKeys are the tag keys describing the grouping of this view.
	// A single Row will be produced for each combination of associated tag values.
```

So that had me question if this is the correct way to name these keys? Are they supposedly getting indexed by the processor?

Reply:
This is to ensure the metrics get tagged like internal metrics do. There is a [TagProcessorKey](https://github.com/open-telemetry/opentelemetry-collector/blob/main/internal/obsreportconfig/obsmetrics/obs_processor.go#L37) in the collector but it's in an internal package so I mocked it out in my metrics.

Reply:
Ahh gotcha and thats where the processorID comes in for the "indexing" for that. Sweet thanks for explaining that 👍